akka-persistence/src/main/scala/akka/persistence/PersistentView.scala

```diff
@@ -88,7 +88,10 @@ trait PersistentView extends Actor with Snapshotter with Stash with StashFactory
 
   private val extension = Persistence(context.system)
   private val viewSettings = extension.settings.view
-  private lazy val journal = extension.journalFor(persistenceId)
+
+  // Use same plugins as view origin actor.
+  private[persistence] lazy val journal = extension.journalFor(persistenceId)
```

Comment:
It must be possible to define `journalPluginId` and `snapshotPluginId` for a PersistentView also, in the same way as for a PersistentActor.

This will not work anyway, e.g. when PersistentActor is on one machine and the PersistentView on another machine.